src/Czechia/README.md

```diff
@@ -1,33 +1,70 @@
-# Mews.Fiscalizations.Czechia (EET)
-
-EET stands for Elektronická Evidence Tržeb, which is Czech version of Fiscal Printers.
+<p align="center">
+    <a href="https://mews.com">
+        <img alt="Mews" src="https://user-images.githubusercontent.com/51375082/120493257-16938780-c3bb-11eb-8cb5-0b56fd08240d.png">
+    </a>
+    <br><br>
+    <b>Mews.Fiscalizations.Czechia (EET)</b> is a .NET library that was built to help reporting of e-invoices to the Czech authorities (EET - Elektronická Evidence Tržeb).
+    <b>Current supported version is 3.0.</b>
+    <br><br>
+    <a href="https://www.nuget.org/packages/Mews.Fiscalizations.Czechia/">
+        <img src="https://img.shields.io/nuget/v/Mews.Fiscalizations.Czechia">
+    </a>
+    <a href="https://github.com/MewsSystems/fiscalizations/blob/master/LICENSE">
+        <img src="https://img.shields.io/github/license/MewsSystems/fiscalizations">
+    </a>
+    <a href="https://github.com/MewsSystems/fiscalizations/actions/workflows/build-and-test-czechia-windows.yml">
+        <img src="https://img.shields.io/github/workflow/status/MewsSystems/fiscalizations/Build%20and%20test%20-%20Czechia%20(Windows)/master?label=windows%20build">
+    </a>
+    <a href="https://github.com/MewsSystems/fiscalizations/actions/workflows/build-and-test-czechia-linux.yml">
+        <img src="https://img.shields.io/github/workflow/status/MewsSystems/fiscalizations/Build%20and%20test%20-%20Czechia%20(Linux)/master?label=linux%20build">
+    </a>
+    <a href="https://www.etrzby.cz/assets/cs/prilohy/EET_pristupove_provozni_informace_playground_3.1.pdf">
+        <img src="https://img.shields.io/badge/v3.0-EET-lightgrey">
+    </a>
+</p>
+
+## 📃 Description
+
+EET stands for Elektronická Evidence Tržeb, which is the Czech version of Fiscal Printers.
 It's an online API provided by the Ministry of Finance in a form of a SOAP Web Service.
 
-## Key features
-- No Czech abbreviations.
-- Early data validation.
-- Intuitive immutable DTOs.
-- SOAP communication (including WS-Security signing).
-- [PKP](doc/data.md) and [BKP](doc/data.md) security code computation.
-- Support for parallel async requests.
-- Logging support
+## ⚙️ Installation
+
+The library can be installed through NuGet packages or the command line as mentioned below:
+```bash
+Install-Package Mews.Fiscalizations.Czechia
+```
 
-## Known issues
+## 🎯 Features
+
+-   Functional approach via [FuncSharp](https://github.com/siroky/FuncSharp).
+-   No Czech abbreviations.
+-   Early data validation.
+-   Asynchronous I/O.
+-   All endpoints are covered with tests.
+-   Intuitive immutable DTOs.
+-   Pipelines that run on both Windows and Linux operating systems.
```

Comment:
I wouldn't list pipelines here, they're not a feature of the library. They are a tool we're using for development of the lib.